doc/tutorial_ti_cc3200.md

```diff
@@ -105,152 +105,112 @@ _You should have a Xively account already created, but if you do not, register o
 
 To have a device communicate through Xively we will first need to tell the Xively system that a device exists. [Log into the Xively CPM app](https://app.xively.com/) to complete the following steps.
 
-1. Create a device template.
- _This device template will represent the CC3200 type board that we are using for this example._
- - Click on `Devices` > `Device templates`
- - Click on `Add  new device template`
- - Enter any name you want (ex: "CC3200 Launchpad") and click `Ok`
+1. Create a CC3200 device representation using the Product Launcher
+_This operation will create device template and a device that will represent the CC3200 type board that we are using for this example._
```

Comment:
This operation will create a device template and a device instance inside the Xively Service to represent your CC3200 board.